Microsoft.Toolkit.Uwp.UI.Controls/LayoutTransformer/LayoutTransformer.cs

```diff
@@ -0,0 +1,537 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
```

Comment:
Can we incorporate this code like this and re-license it as it's from the SO post?